src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 5.3//EN"
+        "http://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
```

Comment:
is 3306 used by default?

Reply:
yes
![image](https://user-images.githubusercontent.com/56148428/94773849-48d08d80-03c5-11eb-8ea5-b7f7db67cc99.png)